### PR TITLE
Configurable precision

### PR DESCRIPTION
**Pull request by cpennington**
This allows the user to set `--precision X` on the commandline to store up to that many digits of precision in response times.

### CODE REVIEW

File: locust/stats.py [hunk 1]
```diff
@@ -157,22 +169,25 @@ def _log_response_time(self, response_time):
...
-        # so that 147 becomes 150, 3432 becomes 3400 and 58760 becomes 59000
-        if response_time < 100:
-            rounded_response_time = response_time
-        elif response_time < 1000:
-            rounded_response_time = int(round(response_time, -1))
-        elif response_time < 10000:
-            rounded_response_time = int(round(response_time, -2))
-        else:
-            rounded_response_time = int(round(response_time, -3))
+        rounded_response_time = self._round_response_time(response_time)
```
Comment (BenjiLee) [inline locust/stats.py]: Our of curiosity, what was the reasoning of this doc? Why not just give the more precise number in the first place?

**Comment by BenjiLee** [comment]
:+1:

**Comment by cpennington** [comment]
@cgoldberg: I just rebased this, so it should be merge-able now.

**Comment by cpennington** [comment]
Is this something that you just don't want? Or were you assuming since it was old it was abandoned? I'd be happy to rebase it to get it mergeable.

**Pull request closed without merging**